data/posts/2022-11-04-considerations-before-first-nomad-trip.md

```diff
@@ -0,0 +1,77 @@
+
+## Entry and Exit Requirements
+
+Before you book a ticket, make sure you understand the entry and exit requirements for that country. Some countries may require visitors from certain countries to get a visa, while other visitors from different countries can enter visa free. The process for visas varies widely by country. Some countries will allow you to pay for the visa ahead of time and issue you one in a matter of days, or if you pay for an expedited one, hours. Others may require you to travel to a regional consulate that might not be particularly close to where you reside to apply for the visa, and then you might have to return on a different day to pick up the visa. You may be required to show certain paperwork, like your boarding pass, when entering a country. Looking into this is very important so that you don't miss out on your trip.
+
+When leaving a country, you may need to show your visa, so make sure you keep it in a safe place. If you want to travel to other countries, you might also want to take a look at those entry requirements as well. For instance, if you were visiting Thailand and interested in visiting some neighboring countries like Vietnam, getting a visa for Vietnam might require that you have additional passport photos -- you may want to have those printed ahead of time. Another thing to keep in mind is that some countries require proof of an onward ticket -- if you don't have one, airlines may refuse to allow you to board, so check into this in order to make sure you are prepared before arriving at the airport and don't have to hastily book a ticket.
+
+## What to Bring
+
+This varies quite a bit. I am not a carry-on only packer -- for me, the stress versus thebenefits aren't really there. Lighter is better, but you do want to make sure you have the stuff you need. Often, you don't need to bring a huge amount of clothes, but if the clothing or shoe sizes you wear are not easy to find in the locations you are considering visiting, make sure to bring enough of those items in case you can't obtain them with ease locally. Make sure to bring medications ahead of time -- I would recommend bringing some sort of pain relief pills as well as digestive medicines so you have them ahead of time -- and fill any necessary prescriptions before you leave. In addition to your computer, phone, and charger, you may want to consider a set of noise-canceling headphones, and I would definitely suggest you bring some external battery packs that could be used to charge your devices on the go.
```

Comment:
```suggestion
This varies quite a bit. I am not a carry-on only packer -- for me, the stress versus the benefits aren't really there. Lighter is better, but you do want to make sure you have the stuff you need. Often, you don't need to bring a huge amount of clothes, but if the clothing or shoe sizes you wear are not easy to find in the locations you are considering visiting, make sure to bring enough of those items in case you can't obtain them with ease locally. Make sure to bring medications ahead of time -- I would recommend bringing some sort of pain relief pills as well as digestive medicines so you have them ahead of time -- and fill any necessary prescriptions before you leave. In addition to your computer, phone, and charger, you may want to consider a set of noise-canceling headphones, and I would definitely suggest you bring some external battery packs that could be used to charge your devices on the go.
```